exercises/practice/promises/.docs/instructions.md

```diff
@@ -1,11 +1,30 @@
 # Instructions
 
-Given the basic `Promise`, implement the following functions:
+The two objectives of this exercise are :
+
+1. Implement a `promisify` function that turns a function using the "callback pattern" into a function that returns a `Promise`. See the example below.
+
+```javascript
+function fetchProduct(productId, function(error, data) {
+    if (error) {
+        // Handle the error
+    } else {
+        // Make something with your data
+    }
+})
+
+const fetchProductAsPromise = promisify(fetchProduct);
+
+// Now you got a function `fetchProductAsPromise`
+// that returns a promise
+fetchProductAsPromise(productId)
+    .then((data) => {})
+    .catch((error) => {});
+```
+
+2. Re-implement the following built-ins `Promise` methods (without using them)
```

Comment:
Should we update the tests to make sure the pupils are not using the built-in methods?

Reply:
We usually don't focus on testing that a student does not cheat so I don't think this should be a priority right now.